model/actors/miner/sector.go

```diff
@@ -12,11 +12,42 @@ import (
 	"github.com/filecoin-project/lily/model"
 )
 
-type MinerSectorInfo struct {
-	Height    int64  `pg:",pk,notnull,use_zero"`
-	MinerID   string `pg:",pk,notnull"`
-	SectorID  uint64 `pg:",pk,use_zero"`
-	StateRoot string `pg:",pk,notnull"`
+// MinerSectorInfoV7Plus is the default model exported from the miner actor extractor.
+// the table is returned iff the miner actor code is greater than or equal to v7.
+// The table receives a new name since we cannot rename the miner_sector_info table, else we will break backfill.
+type MinerSectorInfoV7Plus struct {
+	//lint:ignore U1000 tableName is a convention used by go-pg
+	tableName struct{} `pg:"miner_sector_infos_v7plus"`
```

Comment:
I'm not a fan of this new name!  What was wrong with `miner_sector_infos_v7` where v7 indicates the network version that it was introduced? 

`miner_sector_infos_v7plus` seems like painting ourselves into a box. What does it mean if we introduce another change for network version 9?